includes/reader-activation/sync/class-woocommerce.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * Reader Activation Sync WooCommerce.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Reader_Activation\Sync;
+
+use Newspack\Reader_Activation\Sync;
+use Newspack\Donations;
+use Newspack\WooCommerce_Connection;
+use Newspack\WooCommerce_Order_UTM;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WooCommerce Class.
+ */
+class WooCommerce extends Sync {
```

Comment:
Same as Metadata. I don't see why extend

Reply:
Removed in d22dd80cf82f03ebe88b42d7f811cc7a0eacaefc